

// Mock component that throws an error
const ProblemChild = () => {
  throw new Error('Error thrown from problem child');
};

export default ProblemChild;